components/ToastMessage/components/InfoToast.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { connectStyle } from '@shoutem/theme';
+import BaseToast from './BaseToast';
+
+function InfoToast({ style, props, isVisible }) {
+  return (
+    <BaseToast customToastStyle={style} isVisible={isVisible} {...props} />
```

Comment:
```suggestion
function InfoToast({ style, isVisible, ...otherProps }) {
  return (
    <BaseToast customToastStyle={style} isVisible={isVisible} {...otherProps} />
```

Just a suggestion, I've seen this notation elsewhere and it makes more sense to me because `style` and `isVisible` are also props, so having them next to a prop called `props` seems unusual.

Reply:
props here is actually a named prop, coming from the toast component. Can't do much here other than renaming for more clarity